src/content/docs/apm/agents/go-agent/get-started/go-agent-eol-policy.mdx

```diff
@@ -26,6 +26,51 @@ Any versions not listed in the following table are no longer supported. Please [
     </tr>
   </thead>
 
+    <tbody>
+    <tr>
+      <td>
+        v3.19.1
+      </td>
+
+      <td>
+        Sept 16, 2022
+      </td>
+
+      <td>
+        Sept 16, 2023
+      </td>
+    </tr>
+
+    <tbody>
```

Comment:
@iamemilio I removed some extra <tbody> tags that were causing trouble with the build.